src/README.md

```diff
@@ -152,28 +152,26 @@ _Make sure you have generated the ebooks PDFs first in the main branch, by runni
 gcloud init
 ```
 
-3. Stage the changes locally:
-
-```
-git checkout production
-git status
-git pull
-git pull origin main
-git push
-```
-
-  - Check out the `production` branch
-  - Run `git status` to ensure you don't have any uncommitted changes locally
-  - Merge any remote changes (both origin/production and origin/main branches)
-  - Push the merge-commit back up to origin/production
-
-4. Browse the website locally as one final QA test, then deploy the changes live:
+3. Deploy the site:
```

Comment:
Is "_Make sure you have generated the ebooks PDFs first in the main branch, by running the Generate Ebooks GitHub Action_" above still true?

Reply:
Yes that is still the case as discussed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1230#discussion_r475244806